F1WM/DatabaseModel/Race.cs

```diff
@@ -22,8 +24,9 @@ public class Race
 		public virtual Track Track { get; set; }
 		public virtual Country Country { get; set; }
 		public virtual IEnumerable<Grid> Grids { get; set; }
+        public virtual Result Result { get; set; }
```

Comment:
Race has multiple Results, so this mapping is truly error-prone. Please change the type to IEnumerable<Result> and the property name accordingly.